src/routing/Routes.tsx

```diff
@@ -111,23 +111,23 @@ export default class Routes extends React.Component<Props, State> {
             <Route exact path={locations.activity()} component={ActivityPage} />
             <Route exact path={locations.settings()} component={SettingsPage} />
             <Route exact path={locations.sceneDetail()} component={SceneDetailPage} />
-            {env.get('REACT_APP_FF_ENS')
-              ? [
-                  <Route exact path={locations.ens()} component={ENSListPage} />,
-                  <Route exact path={locations.claimENS()} component={ClaimENSPage} />,
-                  <Route exact path={locations.landSelectENS()} component={LandSelectENSPage} />,
-                  <Route exact path={locations.landAssignENS()} component={LandAssignENSPage} />,
-                  <Route exact path={locations.ensSelectLand()} component={ENSSelectLandPage} />
-                ]
-              : null}
-            {env.get('REACT_APP_FF_WEARABLES')
-              ? [
-                  <Route exact path={locations.avatar()} component={AvatarPage} />,
-                  <Route exact path={locations.itemDetail()} component={ItemDetailPage} />,
-                  <Route exact path={locations.collectionDetail()} component={CollectionDetailPage} />,
-                  <Route exact path={locations.itemEditor()} component={ItemEditorPage} />
-                ]
-              : null}
+            {env.get('REACT_APP_FF_ENS') ? (
+              <>
+                <Route exact path={locations.ens()} component={ENSListPage} />
+                <Route exact path={locations.claimENS()} component={ClaimENSPage} />
+                <Route exact path={locations.landSelectENS()} component={LandSelectENSPage} />
+                <Route exact path={locations.landAssignENS()} component={LandAssignENSPage} />
+                <Route exact path={locations.ensSelectLand()} component={ENSSelectLandPage} />
+              </>
+            ) : null}
+            {env.get('REACT_APP_FF_WEARABLES') ? (
+              <>
+                <Route exact path={locations.avatar()} component={AvatarPage} />
+                <Route exact path={locations.itemDetail()} component={ItemDetailPage} />
+                <Route exact path={locations.collectionDetail()} component={CollectionDetailPage} />
+                <Route exact path={locations.itemEditor()} component={ItemEditorPage} />
+              </>
+            ) : null}
```

Comment:
we should keep this as arrays (`[]`) because this breaks all wearable links (try entering the wearable tab)!

Reply:
Oops, I made this because the array [ ] solution give me linting errors. Do you know why are not working properly? I'll check it.